modules/swagger-codegen/src/main/resources/typescript-fetch/configuration.mustache

```diff
@@ -8,5 +11,6 @@ export class Configuration {
     };
     username: string;
     password: string;
-    accessToken: string;
+    accessToken: string | ((name: string) => string);
+    basePath: string;
 }
```

Comment:
- accessToken var >> return current token
- accessToken function >> use for multiple Oauth, parameter auth name
- basePath >> for override base path